Tests/MongoSwiftTests/BSONValueTests.swift

```diff
@@ -27,4 +28,89 @@ final class BSONValueTests: XCTestCase {
         expect(try Binary(data: sixteenBytes, subtype: .uuidDeprecated)).toNot(throwError())
         expect(try Binary(data: sixteenBytes, subtype: .uuid)).toNot(throwError())
     }
+
+    func bsonEqual(expectedValue: BSONValue?) -> Predicate<BSONValue> {
+        return Predicate { (actualExpression: Expression<BSONValue>) throws -> PredicateResult in
+            let msg = ExpectationMessage.expectedActualValueTo("equal <\(String(describing: expectedValue)))>")
+            if let actualValue = try actualExpression.evaluate() {
+                return PredicateResult(
+                    bool: try bsonEquals(lhs: actualValue, rhs: expectedValue!),
+                    message: msg
+                )
+            } else {
+                return PredicateResult(
+                    status: .fail,
+                    message: msg.appendedBeNilHint()
+                )
+            }
+        }
+    }
+
+    func checkTrueAndFalse(val: BSONValue, alternate: BSONValue) {
```

Comment:
I'm not sure if we're strict about this across our test suite, but should we stick to `fileprivate` or some other limited visibility here? Likely also for `bsonEqual()` above, too.

Reply:
I think this is a good idea. Reducing the visibility of functions like these that are only ever used here should only help us out. I believe I just genuinely missed these when going over the added functions and adding visibilities, so thanks for this catch.

This is done in 69c9dea.